en/docs/chapter_heap/summary.md

```diff
@@ -2,16 +2,16 @@
 
 ### Key review
 
-- A heap is a complete binary tree, which can be divided into a max heap and a min heap based on its property. The top element of a max (min) heap is the largest (smallest).
+- A heap is a complete binary tree, which can be divided into a max heap and a min heap based on its building property. The top element of a max (min) heap is the largest (smallest).
 - A priority queue is defined as a queue with dequeue priority, usually implemented using a heap.
 - Common operations of a heap and their corresponding time complexities include: element insertion into the heap $O(\log n)$, removing the top element from the heap $O(\log n)$, and accessing the top element of the heap $O(1)$.
```

Comment:
In this sentence we use the plural possessive 'their' but we are referring to 'a heap' (which is singular). Change to either
- Common operations of a heap and its corresponding time complexities include: 
- Common operations of heaps and their corresponding time complexities

In my opinion, I think the first option is better.